dora/shell/src/main/java/alluxio/cli/fsadmin/nodes/RemoveWorkerCommand.java

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Should we also make changes to [nodes.go](https://github.com/TachyonNexus/enterprise/blob/main/cli/src/alluxio.org/cli/cmd/info/nodes.go)?

Reply:
yes Rico is making the change

Reply:
added now with the running format:
bin/alluxio process remove-worker -n <worker_id>